src/node/internal/crypto_util.ts

```diff
@@ -0,0 +1,20 @@
+
+/* todo: the following is adopted code, enabling linting one day */
+/* eslint-disable */
+
+import {
+  validateString,
+} from 'node-internal:validators';
+
+export const kHandle = Symbol('kHandle');
+
+export function getDefaultEncoding() : string {
+  return 'buffer';
+}
+
+export function getStringOption(options: any, key: string) : any {
+    let value;
+    if (options && (value = options[key]) != null)
+      validateString(value, `options.${key}`);
+    return value;
+}
```

Comment:
New line at the end of the file
```suggestion
}

```